Index: src/Streams/CMakeLists.txt
===================================================================
--- src/Streams/CMakeLists.txt	(revision 84050)
+++ src/Streams/CMakeLists.txt	(working copy)
@@ -46,7 +46,7 @@
   Time
   RootUtils
   Segments
-  ${GSL_LIBRARIES}
+  ${GSL_LIBRARY}
   ${FRAMEL_LIBRARIES}
   )
 set_target_properties(
Index: src/Triggers/CMakeLists.txt
===================================================================
--- src/Triggers/CMakeLists.txt	(revision 84050)
+++ src/Triggers/CMakeLists.txt	(working copy)
@@ -57,7 +57,7 @@
 target_link_libraries(
   Triggers
   ${HDF5_CXX_LIBRARIES}
-  ${GSL_LIBRARIES}
+  ${GSL_LIBRARY}
   Streams
   )
 set_target_properties(
