diff --git a/src/Streams/CMakeLists.txt b/src/Streams/CMakeLists.txt
index 5bd8f5e..6a6b029 100644
--- a/src/Streams/CMakeLists.txt
+++ b/src/Streams/CMakeLists.txt
@@ -63,6 +63,7 @@ add_executable(
 target_link_libraries(
   gwl-print-channels
   ${ROOT_Core_LIBRARY}
+  ${ROOT_MathCore_LIBRARY}
   Streams
   )
 target_include_directories(
@@ -78,6 +79,7 @@ add_executable(
 target_link_libraries(
   gwl-print-psd
   ${ROOT_Core_LIBRARY}
+  ${ROOT_MathCore_LIBRARY}
   ${ROOT_RIO_LIBRARY}
   Streams
   )
@@ -119,6 +121,7 @@ add_executable(
   )
 target_link_libraries(
   gwl-test-ffl
+  ${ROOT_MathCore_LIBRARY}
   Streams
   )
 
