Index: src/Triggers/CMakeLists.txt
===================================================================
diff --git a/GWOLLUM/trunk/src/Triggers/CMakeLists.txt b/GWOLLUM/trunk/src/Triggers/CMakeLists.txt
--- a/GWOLLUM/trunk/src/Triggers/CMakeLists.txt	(revision 83816)
+++ b/GWOLLUM/trunk/src/Triggers/CMakeLists.txt	(working copy)
@@ -57,7 +57,7 @@
 target_link_libraries(
   Triggers
   ${HDF5_CXX_LIBRARIES}
-  ${GSL_LIBRARIES}
+  ${GSL_LIBRARY}
   Streams
   )
 set_target_properties(
